packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -274,6 +282,20 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
 
       return this[$model]![$materialFromPoint](scene.raycaster);
     }
+
+    uvFromPoint(pixelX: number, pixelY: number): Vector2D|null {
+      const scene = this[$scene];
+      const ndcCoords = scene.getNDC(pixelX, pixelY);
+      scene.raycaster.setFromCamera(ndcCoords, scene.getCamera());
```

Comment:
Would you mind shifting this over next to `positionAndNormalFromPoint` in annotations.ts instead? I think following that logic will also clean this up a shade (no need to access the raycaster out here).

Reply:
Yes I was doing this in the beginning, but then I was not sure if it really fits under "annotations" since I couldn't see a use case relating the hotspots. Though I changed that now.